local/bin/py/build/actions/integrations.py

```diff
@@ -513,6 +513,42 @@ def replace_image_src(markdown_string, integration_name):
         else:
             return markdown_string
 
+    @staticmethod
```

Comment:
will iterate on this solution, for now it needs to handle cases where the pricing header is duplicated in integration readme